py-aioitertools: New package

a-saitoh-fj opened the pull request:
py-aioitertools: New package

a-saitoh-fj committed:
py-aioitertools: New package

adamjstewart commented on var/spack/repos/builtin/packages/py-aioitertools/package.py:
```diff
@@ -0,0 +1,24 @@
+    """Implementation of itertools, builtins, and more for AsyncIO and mixed-type iterables."""
+
+    homepage = "https://aioitertools.omnilib.dev/en/stable/"
+    url      = "https://files.pythonhosted.org/packages/32/0b/3260ac050de07bf6e91871944583bb8598091da19155c34f7ef02244709c/aioitertools-0.7.1-py3-none-any.whl"
```
Can you switch to the source build?

adamjstewart commented on var/spack/repos/builtin/packages/py-aioitertools/package.py:
```diff
@@ -0,0 +1,24 @@
+    version('0.7.1' , sha256='8972308474c41ed5e0636819f948ebff32f2318e70f7e7d23cd208c4357cc773', expand=False)
+
+    extends('python')
+    depends_on('python@3.6:3.7.999', type=('build', 'run'))
```
```suggestion
    depends_on('python@3.6:', type=('build', 'run'))
```

a-saitoh-fj committed:
py-aioitertools: Fixed python dependencies, and Fixed style of code

a-saitoh-fj commented:
Fixed the pointed out point.
- Fixed python dependencies.

And,
I'm sorry, I didn't have enough confirmation.
- The description was too long, so I put a line break
- I deleted it because there was extra space.

"Can you switch to the source build?"
I also tried a source build with tar.gz, but I couldn't build this oss because setup.py isn't available.

adamjstewart commented:
If I download https://files.pythonhosted.org/packages/dc/a0/f149e54ea082e5f4ae71c68e8e72cab8749283329a7c2747336ef7b68dc0/aioitertools-0.7.1.tar.gz and extract it, there is a `setup.py` file right at the top level.

a-saitoh-fj committed:
switch to the source build

a-saitoh-fj commented:
py-aioitertools: Switch to the source build

I'm sorry. The source code I was referring to was different.
I changed to the source build, so please check here.
